liquibase-core/src/test/resources/liquibase/integration/ant/ChangeLogDirectoryTest.xml

```diff
@@ -88,12 +88,14 @@
 	</target>
 
 	<target name="testRelativeChangeLogDirectoryStartsWithSlash">
-        <au:expectfailure expectedmessage="Unable to update database: The file simple-changelog.xml was not found">
-			<lb:updateDatabase
-					changelogdirectory="/src/test/resources/liquibase/integration/ant/changelog"
-					changelogfile="simple-changelog.xml">
-				<lb:database refid="test-db" />
-			</lb:updateDatabase>
+		<au:expectfailure>
+			<au:logcontains text="Unable to rollback database: The file changelog-with-rollback.xml was not found">
```

Comment:
Does anyone think this presents a problem? With the command refactoring we end up throwing an exception with the same message, but the cause changes because we are throwing the exception during validation.

Reply:
I had to change some messages too. I believe it's fine.